### PR TITLE
Improve package.json

### PR DESCRIPTION
**Pull request by nonomoho**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,12 +2,25 @@
...
   "description": "Pattern Library tool",
-  "main": "src/index.pug",
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/tikui/tikui.git"
+  "keywords": [
+    "ui",
+    "kit ui",
+    "design",
```
Comment (Gnuk) [inline package.json]: may be design system

File: package.json [hunk 2]
```diff
@@ -2,12 +2,25 @@
...
+    "web pattern library"
```
Comment (Gnuk) [inline package.json]: maybe web not really needed

File: package.json [hunk 3]
```diff
@@ -2,12 +2,25 @@
...
   "version": "1.0.0",
   "description": "Pattern Library tool",
-  "main": "src/index.pug",
-  "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/tikui/tikui.git"
+  "keywords": [
+    "ui",
+    "kit ui",
```
Comment (Gnuk) [inline package.json]: maybe ui kit

File: package.json (hunk 1 above)
Comment (nonomoho) [reply package.json]: 👍

**Pull request merged**